common/src/main/scala/co/datamechanics/delight/common/package.scala

```diff
@@ -2,4 +2,4 @@ package co.datamechanics.delight
 
 import org.apache.spark.internal.Logging
 
-package object common extends Logging {}
\ No newline at end of file
+package object common extends Logging {}
```

Comment:
What's the purpose of `common` to extend `Logging`? Can't think of a use case for this (and can't wait to learn a new trick in Scala).

Reply:
This `extends` is a bit of legacy that stays after the package split
Thanks to have noticed it, i'll remove it